.sobelow-conf

```diff
@@ -0,0 +1,13 @@
+[
```

Comment:
This is the default file generated by `mix sobelow --save-config` with a little formatting tweak.